woocommerce/inc/wc-enqueue.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * WooCommerce enqueue scripts
+ *
+ * @package Bootscore
+ * @version 5.3.3
+ */
+
+
+// Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
+
+
+/**
+ * Enqueue scripts
+ */
+function wc_scripts() {
+  
+  // Enable fragments script (disabled in WooCommerce 7.8.0 if mini-cart widget is not in a widget position)
+  // See https://developer.woocommerce.com/2023/05/24/woocommerce-7-8-beta-1-released/
+  wp_enqueue_script('wc-cart-fragments');
+
+  // Get modification time. Enqueue files with modification date to prevent browser from loading cached scripts and styles when file content changes. 
+  $modificated_WooCommerceJS = date('YmdHi', filemtime(get_template_directory() . '/woocommerce/js/woocommerce.js'));
+
+  // WooCommerce JS
+  wp_enqueue_script('woocommerce-script', get_template_directory_uri() . '/woocommerce/js/woocommerce.js', array(), $modificated_WooCommerceJS, true);
```

Comment:
Same as the function, adding bootscore to the handle could keep it clear it's from the theme when checking loaded resources.

Reply:
What do you mean exactly here?

Reply:
Got it, done.

https://github.com/bootscore/bootscore/pull/555/commits/d4f5895e1cbdb9bb19c92c60d979a3ed1293a6f6